build-support/docker/travis_ci/Dockerfile

```diff
@@ -25,9 +25,12 @@ RUN chown -R ${TRAVIS_USER} ${TRAVIS_WORK_DIR_PATH}
 
 USER ${TRAVIS_USER}:${TRAVIS_GROUP}
 
-# Our newly created user is unlikely to have a sane environment: set a locale at least.
+# Our newly created user is unlikely to have a sensible environment: set a locale at least.
 ENV LC_ALL="en_US.UTF-8"
 
+# Pants's auto-detection fails for the Docker image used in CI.
+ENV PANTS_DYNAMIC_UI=false
```

Comment:
Should this be in `pants.travis-ci.toml` instead?

Reply:
Oh yeah, lol. It is already. But `PANTS_CONFIG_FILES` is ignored by the docker environment.